src/util.ts

```diff
@@ -42,7 +41,7 @@ export const parseInputFiles = (files: string): string[] => {
 
 export const parseConfig = (env: Env): Config => {
   return {
-    github_token: getInput("token") || env.GITHUB_TOKEN || "",
+    github_token: env.GITHUB_TOKEN || env.INPUT_TOKEN || "",
```

Comment:
note all inputs are just env variables prefixed with `INPUT_`. we gain easy unit testability from leveraging this fact